lib/elixir/lib/enum.ex

```diff
@@ -2494,9 +2499,17 @@ defmodule Enum do
   defp enum_to_string(entry) when is_binary(entry), do: entry
   defp enum_to_string(entry), do: String.Chars.to_string(entry)
 
-  defp random_index(n) do
-    :rand.uniform(n + 1) - 1
-  end
+  @spec random_integer(integer, integer) :: integer
```

Comment:
@eksperimental those are rather the exception than the rule. :) Given the current spec is over generic anyway, I would remove it. :)